packages/react-next/src/components/SpinButton/SpinButton.types.ts

```diff
@@ -160,14 +167,14 @@ export interface ISpinButtonProps extends React.HTMLAttributes<HTMLDivElement> {
    * Custom function for providing the classNames for the control. Can be used to provide
    * all styles for the component instead of applying them on top of the default styles.
    */
-  getClassNames?: (
-    theme: ITheme,
-    disabled: boolean,
-    isFocused: boolean,
-    keyboardSpinDirection: KeyboardSpinDirection,
-    labelPosition?: Position,
-    className?: string,
-  ) => ISpinButtonClassNames;
+  // getClassNames?: (
+  //   theme: ITheme,
+  //   disabled: boolean,
+  //   isFocused: boolean,
+  //   keyboardSpinDirection: KeyboardSpinDirection,
+  //   labelPosition?: Position,
+  //   className?: string,
+  // ) => ISpinButtonClassNames;
```

Comment:
hmmm this prop is so weird. i would love to remove this. @khmakoto ?

Reply:
I think I agree with you.